docs/src/pages/demos/pickers/MaterialUIPickers.js

```diff
@@ -0,0 +1,42 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Grid from '@material-ui/core/Grid';
+import withStyles from '@material-ui/styles/withStyles';
+import DateFnsUtils from '@date-io/date-fns';
+import { MuiPickersUtilsProvider, TimePicker, DatePicker } from 'material-ui-pickers';
+
+class MuiPickersDemo extends PureComponent {
+  static propTypes = {
+    classes: PropTypes.object.isRequired,
+  };
+
+  state = {
+    selectedDate: new Date(),
+  };
+
+  handleDateChange = date => {
+    this.setState({ selectedDate: date });
+  };
+
+  render() {
+    const { classes } = this.props;
+    const { selectedDate } = this.state;
+
+    return (
+      <MuiPickersUtilsProvider utils={DateFnsUtils}>
+        <Grid container className={classes.pickersContainer} justify="space-around">
+          <DatePicker label="Date picker" value={selectedDate} onChange={this.handleDateChange} />
+          <TimePicker label="Time picker" value={selectedDate} onChange={this.handleDateChange} />
+        </Grid>
+      </MuiPickersUtilsProvider>
+    );
+  }
+}
+
+const styles = {
```

Comment:
We tend to move this before the component declaration. Since you need information in render about the classNames it makes more sense to have them visually defined before the render method.